haskell-ide-engine.cabal

```diff
@@ -87,6 +87,7 @@ library
                      , safe
                      , sorted-list >= 0.2.1.0
                      , stm
+                     , syb
```

Comment:
Given none of the other functionality relies on syb do we want to do this? It can be quite slow without additional care (see haddock's hyperlinker for an example of how to make it faster by avoiding recursing into all nodes).